core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -444,10 +319,6 @@ private void testDayOfMonthWithUnderscores(FunctionExpression dateExpression, in
 
   @Test
   public void dayOfMonthWithUnderscores() {
-    lenient().when(nullRef.valueOf(env)).thenReturn(nullValue());
```

Comment:
These will probably make my [PR 204](https://github.com/Bit-Quill/opensearch-project-sql/pull/204/files#diff-bdc6fcaa42a4347bba30e99a1f4434587434f56a26941a3b1f2cf6eed4ddcf0a) for `time_format` conflict. Is it safe to checkout this file into my PR, or should I wait until this is merged upstream and rebase before making my PR for `time_format` upstream.

Reply:
It doesn't matter which PR goes first, but we have to rebase second one once first merged and fix conflicts.